packages/strategy-tests/src/lib.rs

```diff
@@ -1120,17 +1118,111 @@ impl Strategy {
                             operations.push(state_transition);
                         }
                     }
-                    // OperationType::ContractCreate(new_fields_optional_count_range, new_fields_required_count_range, new_index_count_range, document_type_count)
-                    // if !current_identities.is_empty() => {
-                    //     DataContract::;
-                    //
-                    //     DocumentType::random_document()
-                    // }
+                    OperationType::ContractCreate(params, doc_type_count)
+                        if !current_identities.is_empty() =>
+                    {
+                        // Create `count` ContractCreate transitions and push to operations vec
+                        for _ in 0..count {
+                            // Get the contract owner_id from loaded_identity and loaded_identity nonce
+                            let identity = &current_identities[0];
+                            let identity_nonce =
+                                identity_nonce_counter.entry(identity.id()).or_default();
+                            *identity_nonce += 1;
+                            let owner_id = identity.id();
+
+                            // Generate a contract id
+                            let contract_id = DataContract::generate_data_contract_id_v0(
+                                owner_id,
+                                *identity_nonce,
+                            );
+
+                            // Create `doc_type_count` doc types
+                            let mut doc_types = Value::Map(Vec::new());
+                            for _ in doc_type_count.clone() {
```

Comment:
no need for clone I think

Reply:
If no clone, then compile error: `&std::ops::Range<u16>` is not an iterator`